### PR TITLE
DOCSP-39388 Add Job Monitoring Estimate Details

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION
- Added additional mention of the new `Review Summary` form.
- Add an additional metric to the monitor a sync job page.
- Removed excessive screenshot from the monitor sync job page.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39388/jobs/monitoring-jobs/#snapshot-stage

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39388/jobs/creating-jobs/#review-and-start-your-job

## JIRA

https://jira.mongodb.org/browse/DOCSP-39388

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=664b649bcf5bec1c2917f884

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/jobs/creating-jobs.txt [hunk 1]
```diff
@@ -174,9 +174,12 @@ Steps
         3. Execute the statements.
         4. Follow any commented manual steps.
 
-  .. step:: Start your job
+  .. step:: Review and start your job
 
-      Click :guilabel:`Start`. 
+      On the :guilabel:`Review Summary` form, you can see details about 
+      your sync job. These details include the type, the number of 
+      tables and rows affected, as well as the estimated data size of the
```
Comment (jocelyn-mendez1) [inline source/jobs/creating-jobs.txt:181]: [minimalism]

```suggestion
      tables and rows affected, and the estimated data size of the 
```

**Review by jocelyn-mendez1 — APPROVED** [review]
Thanks for the updates @ianf-mongodb, LGTM! I just left a small suggestion for your consideration!

**Comment by ianf-mongodb** [comment]
Hi @davidsoulmdb this is ready for review, please let me know if there are any other areas that need an update for this.

**Review by davidsoulmdb — APPROVED** [review]
LGTM

**Pull request merged**